Bottom: bddfb959164cf008bcd0d8be00e14213b4272a73
Top:    a35952ead6921f75b9c971271af3a86b2f2d988d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:09:08 -0500

Refresh of codebase-generator

---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index c14bd7dd..5a366551 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -172,7 +172,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return giturl.owner
+    return '/'.join((giturl.owner, giturl.repo))
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
